PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp

```diff
@@ -706,7 +711,18 @@ bool PluginsAdminDlg::initFromJson()
 #ifdef DEBUG // if not debug, then it's release
 	
 	// load from nppPluginList.json instead of nppPluginList.dll
-	ifstream nppPluginListJson(_pluginListFullPath);
+#	ifdef __MINGW32__
+#		if (__MINGW32_MAJOR_VERSION < 11)
+			wstring_convert<codecvt_utf8<wchar_t>, wchar_t> converter;
+			const string pathStr(converter.to_bytes(_pluginListFullPath.c_str()));
+			ifstream nppPluginListJson(pathStr);
+#		else
+			ifstream nppPluginListJson(_pluginListFullPath.c_str());
+#		endif
+#	else
+		ifstream nppPluginListJson(_pluginListFullPath);
+#	endif
+
```

Comment:
Please use `ifstream nppPluginListJson(wstring2string(_pluginListFullPath, CP_UTF8));` instead of the above lines.